DashAI/back/tasks/tasks_schemas/TabularClassificationTask.json

```diff
@@ -1,19 +1,7 @@
 {
-	"TabularClassificationTask": [
-		{
-			"class": "CSVDataLoader",
-			"name": "CSV Data",
-			"help": "Use CSV files to upload the data. You can use a .csv file or multiple .csv files in a .zip file.",
-			"description": "You can upload your data in a .csv file or in multiple .csv files in a .zip, where you can have the definition of splits in folders as shown above. \n If you only have one file or multiple files without the folder definition for the splits, you can set them up later. \n Make sure that all the CSV files have the same features.",
-			"images": ["/info_images/csv_files.png"]
-		},
-		{
-			"class": "JSONDataLoader",
-			"name": "JSON Data",
-			"help": "Use JSON files to upload the data. You can use a .json file or multiple .json files in a .zip file.",
-			"description": "You can upload your data in a .json file or in multiple .json files in a .zip, where you can have the definition of splits in folders as shown above. \n If you only have one file or multiple files without the folder definition for the splits, you can set them up later. \n The JSON files have to follow the format that specify in the last figure, the data have to be in a key, that contains an array with the samples of the data or rows in JSON objects, where each one have the features of data or columns like key/value pairs. \n Make sure that all the JSON files have the same features and format.",
-			"images": ["/info_images/json_files.png", "/info_images/json_format.svg"]
-		}
-	],
```

Comment:
I like the idea of removing these from here, but we need to save those on another location. They are still useful.

Reply:
I will add it to the dataloaders endpoint PR